homeassistant/components/humidifier/device_trigger.py

```diff
@@ -46,12 +46,9 @@
     cv.has_at_least_one_key(CONF_BELOW, CONF_ABOVE),
 )
 
-TRIGGER_SCHEMA = vol.All(
-    vol.Any(
-        HUMIDIFIER_TRIGGER_SCHEMA,
-        toggle_entity.TRIGGER_SCHEMA,
-    ),
-    vol.Schema({vol.Required(CONF_DOMAIN): DOMAIN}, extra=vol.ALLOW_EXTRA),
```

Comment:
Both schemas referenced above have "domain" as a required key (all 3 paths lead to the same base schema) so this extra requirement is redundant.

Reply:
While they both require a `domain` key in the configuration, they don't require it to be set to `humidifier` which is why we add it here. I think this change should be reverted.

Reply:
Ahhh, THERE we go.  It took me a few minutes to understand your comment!  Yup, that makes sense, reverting.

In case anybody else reads this, the part I missed was that the second part of the `All` schema doesn't just require that CONF_DOMAIN exists, but it requires that it's set to `DOMAIN` (a constant string `humidifier` in this package).